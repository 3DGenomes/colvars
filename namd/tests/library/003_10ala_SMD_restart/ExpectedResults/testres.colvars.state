configuration {
  step           20
  dt 5.000000e-01
}

colvar {
  name rmsd
  x  1.03466918246938e-01
}

harmonic {
  configuration {
    name harmonic1
    centers  2.00000000000000e-01
    centers_raw  2.00000000000000e-01
    accumulatedWork 1.18707661322155e+02
  }
}

