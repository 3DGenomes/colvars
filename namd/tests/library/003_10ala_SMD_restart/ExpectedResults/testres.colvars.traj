# step         rmsd                  fs_rmsd               fa_rmsd                E_harmonic1           x0_rmsd               W_harmonic1          
          10    3.82640484672394e-02  0.00000000000000e+00  6.17359515327606e+02  1.90566385582769e+01  1.00000000000000e-01 3.54824797528161e+01
          11    4.33662654975169e-02 -2.63053274285563e+02  6.66337345024831e+02  2.22002728687370e+01  1.10000000000000e-01 4.21458532030644e+01
          12    4.88935349256412e-02 -3.05729693240212e+02  7.11064650743588e+02  2.52806468768550e+01  1.20000000000000e-01 4.92564997105003e+01
          13    5.47265696202801e-02 -3.51433778138536e+02  7.52734303797199e+02  2.83304466056527e+01  1.30000000000000e-01 5.67838427484723e+01
          14    6.08255065750144e-02 -4.01239730554414e+02  7.91744934249857e+02  3.13430020455155e+01  1.40000000000000e-01 6.47012920909708e+01
          15    6.71915787720546e-02 -4.53746169213949e+02  8.28084212279454e+02  3.42861731313242e+01  1.50000000000000e-01 7.29821342137654e+01
          16    7.39643323579664e-02 -5.06746114124209e+02  8.60356676420336e+02  3.70106805330524e+01  1.60000000000000e-01 8.15857009779687e+01
          17    8.11171394754637e-02 -5.62508342150881e+02  8.88828605245363e+02  3.95008144751209e+01  1.70000000000000e-01 9.04739870304224e+01
          18    8.83572194438220e-02 -6.18809372858622e+02  9.16427805561781e+02  4.19919961403390e+01  1.80000000000000e-01 9.96382650860402e+01
          19    9.58391193919114e-02 -6.72586304925649e+02  9.41608806080886e+02  4.43313571844536e+01  1.90000000000000e-01 1.09054353146849e+02
          20    1.03466918246938e-01 -7.24474945750237e+02  9.65296148348150e+02  4.65931793637167e+01  2.00000000000000e-01 1.18707661322155e+02
