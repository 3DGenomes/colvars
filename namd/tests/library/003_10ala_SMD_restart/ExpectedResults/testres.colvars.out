colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013 in any publication based on this calculation.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # indexFile = "" [default]
colvars: # analysis = off [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 40000 [default]
colvars: # colvarsTrajAppend = off [default]
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = on [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = rmsd
colvars:   Initializing a new "rmsd" component.
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # scalable = on [default]
colvars:     Scalable calculation is not available for group "atoms" with the current configuration.
colvars:       Initializing atom group "atoms".
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID = { BH }
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # enableFitGradients = on
colvars:       Atom group "atoms" defined, 10 atoms initialized: total mass = 120.11, total charge = 0.53.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = ../Common/pi-ideal.pdb
colvars:     # refPositionsCol = "" [default]
colvars:     Enabling "centerReference" and "rotateReference", to minimize RMSD before calculating it as a variable: if this is not the desired behavior, disable them explicitly within the "atoms" block.
colvars:     This is a standard minimum RMSD, derivatives of the optimal rotation will not be computed as they cancel out in the gradients.
colvars:   All components initialized.
colvars:   # width = 0.01
colvars:   # lowerBoundary = 0
colvars:   # lowerWallConstant = 0 [default]
colvars:   # upperBoundary = 0.1
colvars:   # upperWallConstant = 10
colvars:   # upperWall = 0.1 [default]
colvars:   # hardLowerBoundary = on
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = on
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputSystemForce = on
colvars:   # outputAppliedForce = on
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { rmsd }
colvars:   # outputEnergy = on
colvars:   # forceConstant = 1 [default]
colvars:   # centers = { 0 }
colvars:   # targetCenters = { 0.2 }
colvars:   # targetForceConstant = 0 [default]
colvars:   # targetNumSteps = 20
colvars:   # targetNumStages = 0 [default]
colvars:   # targetForceExponent = 1 [default]
colvars:   # outputCenters = on
colvars:   # outputAccumulatedWork = on
colvars:   The force constant for colvar "rmsd" will be rescaled to 10000 according to the specified width.
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (10 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: Re-initialized atom group rmsd:0/0. 10 atoms: total mass = 120.11.
colvars: Restarting from file "test.colvars.state".
colvars:   Restarting collective variable "rmsd" from value: 1.39199
colvars:   Restarting restraint bias "harmonic1".
colvars:   # centers = { 0.1 }
colvars:   # centers_raw = { 0.1 }
colvars:   # accumulatedWork = -2015.75
colvars: ----------------------------------------------------------------------
colvars: The restart output state file will be "testres.colvars.state".
colvars: The final output state file will be "testres.colvars.state".
colvars: Writing to colvar trajectory file "testres.colvars.traj".
colvars: Updating NAMD interface:
colvars: updating atomic data (10 atoms).
colvars: updating group data (0 scalable groups, 0 atoms in total).
colvars: Saving collective variables state to "testres.colvars.state".
